core/src/main/java/com/google/instrumentation/internal/TestClock.java

```diff
@@ -28,14 +28,15 @@
   private static final int NUM_NANOS_PER_SECOND = 1000 * 1000 * 1000;
 
   @GuardedBy("this")
-  private Timestamp currentTime = Timestamp.create(0, 0);
+  private Timestamp currentTime = Timestamp.create(123, 456);
```

Comment:
Done.  I was also wondering if we should remove the no-arg factory method and use a different clock subclass for tests that don't need to set the time.